django-ory-cloud/tests/playwright/django-ory-cloud.spec.ts

```diff
@@ -0,0 +1,77 @@
+import { test, expect, Page } from '@playwright/test'
+import { randomEmail, randomString } from './helpers'
+
+const login = async (page: Page) => {
+  const email = randomEmail()
+  await page.fill('[name="traits.email"]', email)
+  await page.fill('[name="password"]', randomString())
+  await page.click('[value="password"]')
+  return email
+}
+
+test.describe('django-ory-cloud', () => {
+  const url = 'http://localhost:4000/'
+  const name = 'django'
+  {
+    test.describe('django', async () => {
+      test('login and sign up', async ({ page }) => {
```

Comment:
You don't need the outer `test.describe` call anymore. Please also remove the closing counterpart on line 30.

```suggestion
    test('login and sign up', async ({ page }) => {
```